app/models/miq_report/generator/html.rb

```diff
@@ -50,7 +50,9 @@ def build_html_rows(clickable_rows = false)
             end
           end
           save_val = d.data[sortby[0]].to_s
-          group_text = d.data["display_range"] if db == "Chargeback" && sortby[0] == "start_date" # Chargeback, sort by date, but show range
+          # Chargeback, sort by date, but show range
+          is_chargeback_report = Chargeback.subclasses.collect(&:to_s).include?(db)
```

Comment:
@gtanzillo yes certainly much better 👍 => updated 

I will do same thing in other opened PR after this will merged, so that we will have this method presented in code.